internal/cli/iam/organizations/describe_test.go

```diff
@@ -17,28 +17,48 @@
 package organizations
 
 import (
+	"bytes"
 	"testing"
 
 	"github.com/golang/mock/gomock"
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
 	"github.com/mongodb/mongodb-atlas-cli/internal/mocks"
-	"go.mongodb.org/atlas/mongodbatlas"
+	"github.com/stretchr/testify/assert"
+	atlasv2 "go.mongodb.org/atlas/mongodbatlasv2"
 )
 
 func TestDescribe_Run(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	mockStore := mocks.NewMockOrganizationDescriber(ctrl)
+	defer ctrl.Finish()
```

Comment:
This defer call is no longer necessary and I did a PR to remove them all over the repo which this branch seems to bring back due to the nature of long running branches